game.rb

```diff
@@ -8,6 +8,16 @@ def initialize(multiplayer, last_played_at, publish_date)
     @last_played_at = last_played_at
   end
 
+  def to_hash
+    {
+      'id' => id,
+      'multiplayer' => multiplayer,
+      'last_played_at' => last_played_at,
+      'publish_date' => publish_date,
+      'can_be_archived' => can_be_archived?
+    }
+  end
```

Comment:
This looks great my friend, however, I already implemented these functions in my commit, can you please remove it?